hack/local-up-cluster.sh

```diff
@@ -222,6 +222,7 @@ KUBELET_HOST=${KUBELET_HOST:-"127.0.0.1"}
 # By default only allow CORS for requests on localhost
 API_CORS_ALLOWED_ORIGINS=${API_CORS_ALLOWED_ORIGINS:-/127.0.0.1(:[0-9]+)?$,/localhost(:[0-9]+)?$}
 KUBELET_PORT=${KUBELET_PORT:-10250}
+KUBELET_READ_ONLY_PORT=${KUBELET_READ_ONLY_PORT:-10255}
```

Comment:
I would not expect local-up-cluster to re-enable insecure defaults

Reply:
Given the usage of local-up-cluster, I think we can open it. It 's a testing and learning cluster I believe. It 's also convenient for us to see some information of kubelet, sometimes convenient  for debuging. 🤔  @liggitt

Reply:
Having a way to enable it is ok, but making local-up-cluster default to insecure teaches the wrong things and is misleading when developing something

Reply:
ok, I add an option to enable it, we dont open it by default. @liggitt. see updated.

Reply:
would it be simpler to default this to 0 (matching the config file default) and allowing setting it if desired?

Reply:
yeah, more simpler, But I' more likely use approach now.  Because it is default,  we don't need to write it to config file, and maybe mislead user. @liggitt